configs/platforms/windows-2012r2-x64.rb

```diff
@@ -11,10 +11,10 @@
 
   # C:\tools is likely added by mingw, however because we also want to use that
   # dir for vsdevcmd.bat we create it for safety
-  plat.provision_with "mkdir C:/tools"
+  plat.provision_with "mkdir -p C:/tools"
   # We don't want to install any packages from the chocolatey repo by accident
   plat.provision_with "C:/ProgramData/chocolatey/bin/choco.exe upgrade -y chocolatey"
-  plat.provision_with "C:/ProgramData/chocolatey/bin/choco.exe sources remove -name chocolatey"
+  # plat.provision_with "C:/ProgramData/chocolatey/bin/choco.exe sources remove -name chocolatey"
```

Comment:
@jpogran  I think that by removing this, a newer `cmake` version is installed now(3.19.6, previously it was 3.5.2). This is breaking yaml-cpp build on agent-runtime. Would pining `cmake` to `3.5.2` work for bolt?

Reply:
Apologies, this was my mistake. I was supposed to go back to using internal repo only. Will post a fix shortly